src/lightning/fabric/strategies/deepspeed.py

```diff
@@ -312,11 +313,10 @@ def model(self) -> "DeepSpeedEngine":
 
     @override
     def setup_module_and_optimizers(
-        self, module: Module, optimizers: list[Optimizer]
-    ) -> tuple["DeepSpeedEngine", list[Optimizer]]:
-        """Set up a model and multiple optimizers together.
-
-        Currently, only a single optimizer is supported.
+        self, module: Module, optimizers: list[Optimizer], scheduler: Optional[_LRScheduler] = None
+    ) -> tuple["DeepSpeedEngine", list[Optimizer], Optional[_LRScheduler]]:
```

Comment:
This will return `None`, we need to return `Any` here so we can ignore the `scheduler` if it is not provided in input.